### DISCUSSION TITLE
Blender Hangs during renders randomly?

### DISCUSSION THREAD

**Discussion by krigeta** [bnpr/Malt#304] [Q&A] [answered]
Hello, sometimes when I try to render to check the quality with the outlines, the first time the render works fine then the second time it gets stuck, any specific reason or a way to check what is happening with the Blender, or is it an issue?

**Comment by pragma37** [comment, marked as answer]
Hi!
Please, open a bug report and make sure you read the instructions and include the required info:
https://github.com/bnpr/Malt/issues/new/choose

The Q&A section is meant for "how to" questions.
